### PR TITLE
Minor improvements to the README

### PR DESCRIPTION
**Pull request by mtardy**
Two minor additions to the README:
- [Add conference talk from John and Natalia at eBPF day 2022](https://github.com/cilium/tetragon/commit/90cb6d97a4ff57214db3d0a0f955ac7d4a43c106)
- [Add a small pointer to the development setup in the FAQ](https://github.com/cilium/tetragon/commit/647bdd0013200f73360e168e6b033a68c8ef7447)

### CODE REVIEW

**Review by tixxdz — APPROVED** [review]
Minor things, LGTM thank you @mtardy

File: README.md [hunk 1]
```diff
@@ -708,6 +710,8 @@ this is with `git fetch origin/main && git rebase --signoff origin/main`. Then p
 
 [Keeping your cluster safe from attacks with eBPF](https://www.youtube.com/watch?v=agN68U8x1go) - Jed Salazar & Natália Réka Ivánkó, eBPF Summit, 2021
 
+[You and Your Security Profiles; Generating Security Policies with the Help of eBPF](https://www.youtube.com/watch?v=EhQI1qPVb0E) - John Fastabend & Natália Réka Ivánkó, eBPF Day North America, 2022
+
```
[the remark this answers is not in the document]
Comment (mtardy) [reply README.md:714]: Sure! Good idea :)

File: README.md [hunk 2]
```diff
@@ -683,7 +683,9 @@ repository from the `Issuer` and `Subject` fields of the output.
 **Q:** Can I install and use Tetragon in standalone mode (outside of k8s)?
 
 **A:** Yes! You can run `make` to generate standalone binaries and run them directly.
-`make install` will do the same thing and also install to your `PATH`.
+Make sure to take a look at the [Development Setup](docs/contributing/development/README.md#development-setup)
+guide for the build requirements. Also, `make install` will do the same thing
+and also install to your `PATH`.
```
[the remark this answers is not in the document]
Comment (mtardy) [reply README.md:688]: Indeed, a took a look at the `install` target and it seemed incomplete. The instruction on how to run tetragon on standalone is definitely more important here.

**Comment by tixxdz** [comment]
The CI failure is unrelated, merging!

**Pull request merged**